tree/30_generic_methods/condition_from_command.cf

```diff
@@ -67,7 +67,7 @@ bundle agent condition_from_command(condition_prefix, command, true_codes, false
   methods:
       "report"
         usebundle  => _log_v3("Execute the test command ${command} to create ${condition_prefix}_{true,false}", "${condition_prefix}", "${old_class_prefix}", "${class_prefix}", @{args}),
-        ifvarclass => "(!has_promiser_stack.${old_class_prefix}_reached)|(has_promiser_stack.${class_prefix}_reached)";
+        ifvarclass => "${class_prefix}_reached";
```

Comment:
For example in this case, class_prefix may not be defined if command length is almost 4k.

Reply:
I just added an unit test for a 4k command